chapters/ch18.md

```diff
@@ -6,47 +6,53 @@
 
 * If your classifier makes ~20 mistakes on eyeball dev examples, you would start to get a rough sense of the major error sources.
 ->
-
+* Nếu bộ phân loại tạo ra gần 20 lỗi trong tập mẫu phát triển của nhãn cầu, bạn sẽ bắt đầu cảm nhận được sơ bộ về các nguồn lỗi chủ yếu.
 
 * With ~50 mistakes, you would get a good sense of the major error sources.
 ->
-
+* Với gần 50 lỗi, bạn sẽ cảm nhận được các nguồn lỗi chủ yếu một cách khá tốt.
 
 * With ~100 mistakes, you would get a very good sense of the major sources of errors. I’ve seen people manually analyze even more errors—sometimes as many as 500. There is no harm in this as long as you have enough data.
 ->
-
+* Với gần 100 lỗi, bạn sẽ cảm nhận được rất tốt về các nguyên nhân lỗi chính. Tôi đã chứng kiến nhiều người phân tích thủ công tạo ra thậm chí là nhiều tới 500 lỗi. Điều đó không gây hại trong trường hợp bạn có đủ dữ liệu.
 
 Say your classifier has a 5% error rate. To make sure you have ~100 misclassified examples in the Eyeball dev set, the Eyeball dev set would have to have about 2,000 examples (since 0.05*2,000 = 100). The lower your classifier’s error rate, the larger your Eyeball dev set needs to be in order to get a large enough set of errors to analyze.
 ->
 
+341/5000
+Giả sử phân loại của bạn có tỷ lệ lỗi 5%. Để đảm bảo bạn có ~ 100 ví dụ được phân loại sai trong tập dữ liệu phát triển nhãn cầu, tập dữ liệu phát triển nhãn cầu sẽ phải có khoảng 2.000 ví dụ (bởi vì 0,05 * 2.000 = 100). Tỷ lệ lỗi phân loại của bạn càng thấp, tập dữ liệu phát triển nhãn cầu của bạn càng lớn để có được một tập lỗi đủ lớn để phân tích.
 
 If you are working on a task that even humans cannot do well, then the exercise of examining an Eyeball dev set will not be as helpful because it is harder to figure out why the algorithm didn’t classify an example correctly. In this case, you might omit having an Eyeball dev set. We discuss guidelines for such problems in a later chapter.
 ->
-
+Nếu bạn đang thực hiện một nhiệm vụ mà ngay cả con người cũng không thể làm tốt, thì bài tập kiểm tra tập phát triển nhãn cầu sẽ không hữu ích vì khó hình dung tại sao thuật toán không phân loại một ví dụ chính xác. Trong trường hợp này, bạn có thể bỏ qua việc thiết lập tập phát triển nhãn cầu. Chúng tôi thảo luận các hướng dẫn cho những vấn đề như vậy trong một chương sau.
 
 ![img](../imgs/C18_02.png)
 
 How about the Blackbox dev set? We previously said that dev sets of around 1,000-10,000 examples are common. To refine that statement, a Blackbox dev set of 1,000-10,000 examples will often give you enough data to tune hyperparameters and select among models, though there is little harm in having even more data. A Blackbox dev set of 100 would be small but still useful.
 ->
 
+372/5000
```

Comment:
Dòng này là như nào nhỉ?

Reply:
có thể em miss chỗ nào đó khi đọc comment của bác Tiệp và author dịch chương này, nhưng chưa thấy giải thích rõ ý  "Blackbox dev set". Theo e hiểu thì "blackbox dev set" refer đến dataset mà khi tiến hành phân tích lỗi (manually) cũng không hiểu tại sao error ở lỗi đó, e.g., data không phải là images. Vì thế nên với blackbox dev set cần số mẫu lớn (1000 -10000). 
Nếu như trong số các chương trước có nói về blackbox dev set thì bác và Author vui lòng skip comment này của e.

Reply:
e vừa đọc chap 17, thấy nó có address câu hỏi của e ở trên. bác Tiệp và author đọc thì skip comment trên của e nhé.

Reply:
> How about the Blackbox dev set? We previously said that dev sets of around 1,000-10,000 examples are common. To refine that statement, a Blackbox dev set of 1,000-10,000 examples will often give you enough data to tune hyperparameters and select among models, though there is little harm in having even more data. A Blackbox dev set of 100 would be small but still useful.
Em đang hiểu "Blackbox dev set" ở đây giống như tập test ấy. "Blackbox" là những gì mà mình chưa biết, cần kiểm chứng để xem mô hình hoạt động tốt hay không tốt. Em thì hiểu ý tác giả như thế này: Nếu tập blackbox dev set lấy từ 1000-10000 mẫu thì sẽ đủ data để lựa chọn mô hình vì khi đó dữ liệu đã đủ rộng để cover được hầu hết các dạng lỗi. Việc mình thêm nhiều dữ liệu hơn vào "blackbox dev set" để kiểm chứng cũng ít gây ảnh hưởng ("there is little harm") đến đánh giá sai số khi lựa chọn mô hình.